api/src/pcapi/routes/public/collective/endpoints/adage_mock/bookings.py

```diff
@@ -102,16 +102,20 @@ def adage_mock_cancel_collective_booking(booking_id: int) -> None:
     """
     booking = _get_booking_or_raise_404(booking_id)
 
+    # avoid extra query in case of a rollback
+    # booking id can't be saved to avoid an extra query (TODO: why?)
+    api_key_id = current_api_key.id
+
     try:
         reason = models.CollectiveBookingCancellationReasons.PUBLIC_API
         booking_api.cancel_collective_booking(booking, reason=reason, _from="adage_mock_api")
     except exceptions.CollectiveBookingAlreadyCancelled:
         raise ForbiddenError({"code": "ALREADY_CANCELLED_BOOKING"})
     except exceptions.BookingIsAlreadyRefunded:
         raise ForbiddenError({"code": "BOOKING_IS_REIMBURSED"})
-    except Exception as err:
-        err_extras = {"booking": booking_id, "api_key": current_api_key.id, "err": str(err)}
-        logger.error("Adage mock. Failed to cancel booking.", extra=err_extras)
+    except Exception:
+        err_extras = {"booking": booking.id, "api_key": api_key_id}
```

Comment:
Est-ce que ça ne serait pas mieux `api_key_id` ?
```suggestion
        err_extras = {"booking": booking.id, "api_key_id": api_key_id}
```